src/components/TodoList.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <ul>
-            <v-card
-                max-width="950"
-                class="mx-auto">
+  <div>
+    <v-btn @click="onShowForm" color="blue">Add Todo</v-btn><!--button to display dialoge input texts!-->
+             <v-card
+                max-width="950" class="mx-auto">
                 <v-list>
                     <v-list-item v-for="(todo, index) in todos" :key="index">
                         Task: {{ todo.task }}
```

Comment:
I think according to the task requirement, You are asked to do the listing in vuetify data table instead of it. Try to use the same mentioned on task details.